app/src/main/java/org/fossasia/pslab/fragment/HomeFragment.java

```diff
@@ -87,12 +87,10 @@ public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
                 public void run() {
                     try {
 
-                        String version;
-                        PreferenceManager preferenceManager = new PreferenceManager(getActivity());
-                        if ("none".equals(preferenceManager.getVersion())) {
+                        String version = PacketHandler.version;
+                        if ("".equals(version)) {
```

Comment:
[TextUtils.isEmpty](https://developer.android.com/reference/android/text/TextUtils.html#isEmpty(java.lang.CharSequence)), checks if a string is null or 0-length. It can't be null as I have initialised it with 0-length. So I am checking for 0-length only.